ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/disconnected/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-vsphere-upi-disconnected-workers-rhcos-rhel8-provision-chain.yaml

```diff
@@ -6,7 +6,6 @@ chain:
   - ref: workers-rhel-vsphere-provision-dns
   - ref: workers-rhel-pre-hook-haproxy
   - ref: workers-rhel-pre-hook-sshtunnel
-  - ref: workers-rhel-pre-hook-beta-repos
```

Comment:
@gpei any comments to the removal?

Reply:
yeah, this is something we need to do, since RHEL-8.8 is already GA on 2023-05-16

Reply:
@gpei if that, shall we also remove them from aws workflows, right?  cc @yunjiang29 
```
[jianlinliu@jialiu-laptop rehearse]$ grep -r "workers-rhel-pre-hook-beta-repos"
aws/ipi/proxy/workers-rhel8/provision/cucushift-installer-rehearse-aws-ipi-proxy-workers-rhel8-provision-chain.yaml:  - ref: workers-rhel-pre-hook-beta-repos
aws/ipi/proxy/sdn/workers-rhel8/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-provision-chain.yaml:  - ref: workers-rhel-pre-hook-beta-repos
aws/ipi/disconnected/private/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-provision-chain.yaml:  - ref: workers-rhel-pre-hook-beta-repos
aws/ipi/disconnected/private/sdn/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-sdn-workers-rhcos-rhel8-provision-chain.yaml:  - ref: workers-rhel-pre-hook-beta-repos
```

Reply:
right, we're using `PLATFORM_VERSION` to control the RHEL-8.x image being used in AWS workflows, all the four chains are using 8.8, so now we don't need the `workers-rhel-pre-hook-beta-repos` step anymore.

```
[root@preserve-gpei-worker aws]# grep -rl "workers-rhel-pre-hook-beta-repos" |xargs grep PLATFORM_VERSION -A 1
ipi/disconnected/private/sdn/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-sdn-workers-rhcos-rhel8-provision-chain.yaml:  - name: PLATFORM_VERSION
ipi/disconnected/private/sdn/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-sdn-workers-rhcos-rhel8-provision-chain.yaml-    default:  "8.8"
--
ipi/disconnected/private/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-provision-chain.yaml:  - name: PLATFORM_VERSION
ipi/disconnected/private/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-provision-chain.yaml-    default:  "8.8"
--
ipi/proxy/sdn/workers-rhel8/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-provision-chain.yaml:  - name: PLATFORM_VERSION
ipi/proxy/sdn/workers-rhel8/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-provision-chain.yaml-    default:  "8.8"
--
ipi/proxy/workers-rhel8/provision/cucushift-installer-rehearse-aws-ipi-proxy-workers-rhel8-provision-chain.yaml:  - name: PLATFORM_VERSION
ipi/proxy/workers-rhel8/provision/cucushift-installer-rehearse-aws-ipi-proxy-workers-rhel8-provision-chain.yaml-    default:  "8.8"
```

Reply:
Cool~, remove the step together in all aws workflow.